res/values/strings.xml

```diff
@@ -205,6 +210,9 @@
     <string name="media_rewind_description">Rewind button</string>
     <string name="media_play_pause_description">Play or pause button</string>
     <string name="media_forward_description">Fast forward button</string>
+    <string name="button_text_camera"> Camera </string>
```

Comment:
Why are you using spaces before and after the value of this resource? If you want to add a margin or padding, please include it in the xml file instead of here.

Reply:
Are you using button_text_camera resource?